src/main/kotlin/no/nav/faktureringskomponenten/controller/dto/FakturaserieRequestDto.kt

```diff
@@ -49,7 +49,7 @@ data class FakturaserieRequestDto(
         description = "Betalingsintervall",
         example = "KVARTAL",
     )
-    val intervall: FakturaserieIntervall = FakturaserieIntervall.MANEDLIG,
+    val intervall: FakturaserieIntervall = FakturaserieIntervall.KVARTAL,
```

Comment:
Hvor kommer denne endringen fra?

Reply:
Muntlig diskusjon. Det er kvartalsmessig som brukes i avgiftssystemet idag, så det gir mer mening å ha dette som default istedenfor månedlig. Bruker kan heller ikke velge dette i søknaden så muligheten for å velge dette skal fjernes fra vedtak-steget. Det vil i den oppgaven bli viktig at vi har rett default.

Kan reverte dette og vente til endring i vedtaksteget om det blir forvirrende 🤷

Reply:
Hvis du kopierer en kommentar i commiten er det greit for meg